doc/whats_new/v0.22.rst

```diff
@@ -495,6 +496,16 @@ Changelog
   requires less memory.
   :pr:`14108`, :pr:`14170`, :pr:`14296` by :user:`Alex Henrie <alexhenrie>`.
 
+- |Efficiency| :class:`linear_model.RidgeCV` now does not allocate a
+  potentially large array to store dual coefficients for all hyperparameters
+  during its `fit`, nor an array to store all LOO predictions unless
+  `store_cv_values` is `True`.
+  :pr:`15183` by :user:`Jérôme Dockès <jeromedockes>`.
+
+- |Fix| In :class:`linear_model.RidgeCV`, the predicitons reported by
```

Comment:
I think there is a confusion in this whatsnew entry. the problem mentioned is described in **issue** #13998 and is not fixed in this PR

Reply:
Sorry, I saw you fixed it too -- then maybe it is the pr number that needs to change?

Reply:
Let's focus on the PR itself and ignore what's new now.